File: app/src/main/java/com/examle/curexchange/ui/home/FirstCurrencyView.java
package com.examle.curexchange.ui.home;

import com.arellomobile.mvp.viewstate.strategy.AddToEndStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;
import com.examle.curexchange.ui.base.BaseView;

import java.util.List;

@StateStrategyType(AddToEndStrategy.class)
public interface FirstCurrencyView extends BaseView {

    void showData(List<String> name);

    void handleFloatButton(List<String> names, int value);
}

File: app/src/main/java/com/examle/curexchange/ui/home/FirstCurrencyView.kt
package com.examle.curexchange.ui.home

import com.arellomobile.mvp.viewstate.strategy.AddToEndStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType
import com.examle.curexchange.ui.base.BaseView

@StateStrategyType(AddToEndStrategy::class)
interface FirstCurrencyView : BaseView{

    fun showData(names: MutableList<String>)

    fun handleFloatButton(names: MutableList<String>, value: Int)
}